src/wayland-egldisplay.c

```diff
@@ -898,7 +960,7 @@ EGLDisplay wlEglGetPlatformDisplayExport(void *data,
         }
     }
 
-    if (!protocols.hasDrm || (!protocols.hasEglStream && !protocols.hasDmaBuf)) {
+    if (!protocols.hasEglStream && !protocols.hasDmaBuf) {
```

Comment:
Do we really need to change this condition? If we don't have `zwp_linux_dmabuf_v1`  or `wl_eglstream` but we do have `wl_drm` I think we're still fine, are we not?

Reply:
We definitely have to change this because this always results in a failure if wl_drm is not available.
I don't think that we are fine with wl_drm alone. This if statement currently leads to a fail if neither zwp_linux_dmabuf_v1 nor wl_eglstream are available

Reply:
Oh, yeah you're right. Sorry, I should have thought it through more carefully.